File: src/test/java/com/dh/ondot/schedule/fixture/PlaceFixture.java
package com.dh.ondot.schedule.fixture;

import com.dh.ondot.schedule.domain.Place;

public class PlaceFixture {

    public static Place defaultDeparturePlace() {
        return Place.builder()
                .title("집")
                .roadAddress("서울특별시 강남구 테헤란로 1길")
                .latitude(37.5665)
                .longitude(126.9780)
                .build();
    }

    public static Place defaultArrivalPlace() {
        return Place.builder()
                .title("회사")
                .roadAddress("서울특별시 강남구 테헤란로 2길")
                .latitude(37.5000)
                .longitude(127.0000)
                .build();
    }
}

File: src/test/kotlin/com/dh/ondot/schedule/fixture/PlaceFixture.kt
package com.dh.ondot.schedule.fixture

import com.dh.ondot.schedule.domain.Place

object PlaceFixture {

    @JvmStatic
    fun defaultDeparturePlace(): Place = Place(
        0L,
        "집",
        "서울특별시 강남구 테헤란로 1길",
        126.9780,
        37.5665
    )

    @JvmStatic
    fun defaultArrivalPlace(): Place = Place(
        0L,
        "회사",
        "서울특별시 강남구 테헤란로 2길",
        127.0000,
        37.5000
    )
}
